### PR TITLE
test burger game

### PR DESCRIPTION
**Pull request by jim4067**

### CODE REVIEW

File: .vscode/settings.json
Comment (Aiyualive) [inline .vscode/settings.json:1]: this file has been ignored from main branch I believe

File: src/EpplexProvider.ts [hunk 1]
```diff
@@ -380,6 +390,11 @@ class EpplexProvider {
                 token22Program: TOKEN_2022_PROGRAM_ID,
                 systemProgram: SystemProgram.programId,
             })
+            .preInstructions([
```
Comment (Aiyualive) [inline src/EpplexProvider.ts:393]: does this really require 500_000 CUs?

File: src/EpplexProvider.ts [hunk 2]
```diff
@@ -397,12 +412,22 @@ class EpplexProvider {
...
-                tokenProgram: TOKEN_PROGRAM_ID
+                tokenProgram: TOKEN_PROGRAM_ID,
             })
             .transaction();
 
         return tokenBurnTx;
     }
+
+    async getGameConfig(): Promise<GameConfig> {
+        const gameConfig = getGameConfigAccount();
```
Comment (Aiyualive) [inline src/EpplexProvider.ts:423]: we have examples on how to derive PDAs inside the constants folder

File: src/types/EpplexProviderTypes.ts [hunk 3]
```diff
@@ -51,21 +51,22 @@ export interface BurnTxParams {
 }
 
 type GameStatus = IdlTypes<EpplexBurger>["GameStatus"];
```
Comment (Aiyualive) [inline src/types/EpplexProviderTypes.ts:53]: Could move this into a separate folder

File: src/EpplexProvider.ts (hunk 2 above)
Comment (jim4067) [reply src/EpplexProvider.ts:423]: this is for fetching the game config account

**Review by Aiyualive — COMMENTED** [review]
Looks good, just some changes to make sure that tests pass!

File: src/EpplexProvider.ts (hunk 2 above)
Comment (Aiyualive) [reply src/EpplexProvider.ts:423]: right, was referring to the fact that we don't need a separate pda.ts, could follow how we derive the other PDAs

File: src/EpplexProvider.ts (hunk 1 above)
Comment (jim4067) [reply src/EpplexProvider.ts:393]: this pre-instruction should be removed.

File: .vscode/settings.json
Comment (jim4067) [reply .vscode/settings.json:1]: must have re-introduced it when fixing the merge conflict.

File: src/EpplexProvider.ts (hunk 2 above)
Comment (jim4067) [reply src/EpplexProvider.ts:423]: sounds good, let me make the changes.

**Pull request merged**